senza/templates/postgresapp.py

```diff
@@ -208,6 +221,81 @@
             Resource:
               - {{kms_arn}}
           {{/kms_arn}}
+  SpiloMasterSG:
+    Type: "AWS::EC2::SecurityGroup"
+    Properties:
+      GroupDescription: "Security Group for the master ELB"
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: {{postgres_port}}
+          ToPort: {{postgres_port}}
+          CidrIp: {{elb_access_cidr}}
+  {{#add_replica_loadbalancer}}
+  SpiloReplicaSG:
+    Type: "AWS::EC2::SecurityGroup"
+    Properties:
+      GroupDescription: "Security Group for the replica ELB"
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: {{postgres_port}}
+          ToPort: {{postgres_port}}
+          CidrIp: 0.0.0.0/0
```

Comment:
Should we do the same for the replica ELB as we do for the master one? I think securing the database vs unauthorized reads is as important as securing it vs unauthorized writes